File: pulse-android-otel/pulse-utils/src/main/java/com/pulse/utils/PulseLogLevel.kt
package com.pulse.utils

/**
 * SDK log verbosity. Declaration order matches iOS `PulseLogLevel` and JS `PulseLogLevel`
 * (ordinal / numeric value 0 = most verbose, 5 = none).
 */
public enum class PulseLogLevel {
    VERBOSE,
    DEBUG,
    INFO,
    WARN,
    ERROR,
    NONE,
}

File: pulse-web-otel/src/pulse-log-level.ts
/**
 * SDK log verbosity. Numeric order matches React Native and Android Kotlin ordinals
 * (lower = more verbose; {@link PulseLogLevel.NONE} suppresses all SDK diagnostics).
 */
export enum PulseLogLevel {
  VERBOSE = 0,
  DEBUG = 1,
  INFO = 2,
  WARN = 3,
  ERROR = 4,
  NONE = 5,
}
